test/GovernorCountingFractional.t.sol

```diff
@@ -403,6 +404,9 @@ contract GovernorCountingFractionalTest is Test {
       _proposalId, _voter.support, "I have my reasons", new bytes(0), _v, _r, _s
     );
 
+    // Nonce should not have updated.
+    assertEq(_initNonce, governor.nonces(_voter.addr));
```

Comment:
`assertEq(_initNonce, governor.nonces(_voter.addr), "Nonce should not have updated");` is supported and the string shows up in the command line when looking at test failures. Pretty useful.

Reply:
Good idea! Added in 4067f4b